datalad_fuse/tests/test_fuse.py

```diff
@@ -74,7 +77,9 @@ def test_fuse_subdataset(tmp_path, superdataset, cache_clear, transparent, tmp_h
         dots = [".datalad", ".git", ".gitattributes"]
     else:
         dots = [".datalad", ".gitattributes"]
-    with fusing(ds.path, tmp_path, transparent=transparent, wait=True) as mount:
+    with fusing(
+        ds.path, tmp_path, caching=True, transparent=transparent, wait=True
+    ) as mount:
```

Comment:
do you have ideas why codecov claims that we are loosing now coverage  of the `cache-clear`?
https://app.codecov.io/gh/datalad/datalad-fuse/pull/104/blob/datalad_fuse/fsspec_cache_clear.py

Reply:
I don't know.  It doesn't appear we ever tested that command to begin with.

Reply:
<details>
<summary>indeed... odd... I reran with this patch and did not spot traceback from being printed :-/</summary> 

```patch
diff --git a/datalad_fuse/fsspec_cache_clear.py b/datalad_fuse/fsspec_cache_clear.py
index 60fcfbb..10d726b 100644
--- a/datalad_fuse/fsspec_cache_clear.py
+++ b/datalad_fuse/fsspec_cache_clear.py
@@ -1,3 +1,5 @@
+import sys
+import traceback
 from typing import Any, Dict, Iterator, Optional
 
 from datalad.distribution.dataset import (
@@ -42,6 +44,7 @@ class FsspecCacheClear(Interface):
     def __call__(
         dataset: Optional[Dataset] = None, recursive: bool = False
     ) -> Iterator[Dict[str, Any]]:
+        traceback.print_stack(file=sys.stderr)
         ds = require_dataset(
             dataset, purpose="clear fsspec cache", check_installed=True
         )

```
</details>